src/app/content/components/Page/contentDOMTransformations.ts

```diff
@@ -16,43 +16,29 @@ export const transformContent = (document: Document, rootEl: HTMLElement, intl:
   setLinksAttributes(rootEl);
 };
 
-const toggleSolutionSectionStyles = (section: HTMLElement, shouldBeVisible: boolean) => {
-  if (shouldBeVisible) {
-    section.style.height = 'auto';
-    section.style.overflow = 'visible';
-  } else {
-    section.style.height = '0px';
-    section.style.overflow = 'hidden';
-  }
-};
-
 const toggleSolutionAttributes = (solution: HTMLElement, intl: IntlShape) => {
```

Comment:
details/solution is naturally accessible, we shouldn't need to do any of this stuff